### ISSUE TITLE
Package release

### ISSUE THREAD

**Issue by rcmalli** [orobix/quadra#5]
`quadra` needs to be installed using tags defined inside the repository. For example:

```bash
pip install git+https://github.com/orobix/quadra.git@v1.0.0
```

The main reason for this problem is that PyPI does not allow having direct links defined under the `install_required (setup.py)` or `dependencies (pyproject.toml)` list outside of PyPI. Because the Quadra relies on `anomalib` fork maintained under [orobix/anomalib](https://github.com/orobix/anomalib), this activity is on hold until resolved.

Possible solutions:
- Release also `anomalib` package from the fork and add the correct dependency.
- Remove the `anomalib` dependency from the list and installation tutorials mentioning that `anomalib` has to be installed before `quadra`.

**Comment by lorenzomammana** [comment]
Closed by #126

**Comment by lorenzomammana** [comment]
Closed by https://github.com/orobix/quadra/pull/126
